guides/messaging-high-availability-openshift.adoc

```diff
@@ -0,0 +1,296 @@
+= Deploying High-Availability Messaging with WildFly and AMQ 7 on OpenShift
+:summary: Discover how to configure WildFly with AMQ 7 (ActiveMQ Artemis) on OpenShift, using a clustered, high-availability topology for reliable messaging.
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+:prerequisites-time: 20
+
+In this guide, you will learn how to configure a WildFly server connected to a remote AMQ 7 cluster on OpenShift using operators.
+
+include::{includedir}/_prerequisites.adoc[]
+* Access to an OpenShift cluster (consider using the 'Self-managed' variant for a local development environment, available for free at https://developers.redhat.com/products/openshift/download[Red Hat OpenShift, window=_blank])
+* https://docs.openshift.com/container-platform/{ocp-version}/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI, window=_blank] tool
+* link:https://podman.io/[Podman, window="_blank"] (link:https://www.docker.com/[Docker, window="_blank"] is also compatible with minor adjustments)
+
+// login to OpenShift cluster
+include::{includedir}/_proc-log-into-openshift-cluster.adoc[]
+
+== Install and deploy AMQ 7 using an operator
+
+To install the AMQ 7 Operator, follow the instructions in the Red Hat documentation: https://access.redhat.com/documentation/en-us/red_hat_amq_broker/7.11/html-single/deploying_amq_broker_on_openshift/index#proc-br-installing-operator-to-project-from-operatorhub_broker-ocp[Installing the Operator using OperatorHub, window=_blank]
```

Comment:
Shouldn't the target be AMQ 7.12 ?
https://docs.redhat.com/en/documentation/red_hat_amq_broker/7.12/html-single/deploying_amq_broker_on_openshift/index

Reply:
Right, fixed in all occurrences.